frontend/public/components/factory/modal.tsx

```diff
@@ -30,7 +30,7 @@ export const createModalLauncher: CreateModalLauncher = (Component) => (props) =
           isOpen={true}
           contentLabel="Modal"
           onRequestClose={closeModal}
-          className="modal-dialog"
+          className={`modal-dialog ${props.modalClassName}`}
```

Comment:
```suggestion
          className={classNames('modal-dialog', props.modalClassName)}
```

Reply:
In this case, I would simply assign the class to a `div` inside the dialog. You should be able to achieve the same result without needing this extra prop.

Reply:
I will look into this to be sure, but I was trying to utilize the bootstrap class modal-lg which I think needs to be at the same level as modal-dialog to properly adjust the size of the modal.

Reply:
OK makes sense